libraries/AP_Logger/LogStructure.h

```diff
@@ -76,7 +76,9 @@ const struct UnitStructure log_Units[] = {
     { 'W', "Watt" },        // Watt
     { 'Y', "us" },            // pulse width modulation in microseconds
     { 'z', "Hz" },            // Hertz
-    { '#', "instance" }       // (e.g.)Sensor instance number
+    { '#', "instance" },      // (e.g.)Sensor instance number
+    { '0', "ma.h" },          // milliamp hour
```

Comment:
It's a small thing but I think we should skip the "." in the names.  "degC" doesn't have one for example.

Reply:
> It's a small thing but I think we should skip the "." in the names. "degC" doesn't have one for example.

`.` here is being used in the "multiplication" sense rather.  i.e. `ma*h`  

Same doesn't apply for `degC` as it isn't "degrees*Celsius"

Of course, we could just spell it out as "milliamphour"

Reply:
mA.h has my vote. But the designation must not be 0